package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "alfresco-content-app",
-  "version": "4.5.0",
+  "version": "4.4.1",
```

Comment:
Could you please update this file again because you have again here some change which is old, actually we have 4.5.0 version?